FAQ.md

```diff
@@ -124,3 +124,14 @@ for i in ${userlist[@]};do
     MENU_OPTION=1 CLIENT=$i PASS=1 ./openvpn-install.sh
 done
 ```
+---
+
+**Q:** For my clients - I want to set my internal network to pass through the VPN and the rest to go through my internet?
+
+**A:** You would need to edit the `.ovpn` file. You can edit the template out of which those files are created by editing `/etc/openvpn/client-template.txt` file and adding
```

Comment:
In my opinion `client-template.txt` deserves its own question. @angristan

Reply:
Agreed!

Reply:
I agree. Shall I add them?